src/aux_utils.cpp

```diff
@@ -739,15 +743,42 @@ namespace diskann {
         if (reorder_data_file_size != 8 + sizeof(float) *
                                               (size_t) npts_reorder_file *
                                               (size_t) ndims_reorder_file)
-          throw ANNException("Discrepancy in reorder data file size ", -1,
+          throw diskann::ANNException("Discrepancy in reorder data file size ", -1,
                              __FUNCSIG__, __FILE__, __LINE__);
       } catch (std::system_error &e) {
         throw FileException(reorder_data_file, e, __FUNCSIG__, __FILE__,
                             __LINE__);
       }
     }
 
-    // create cached reader + writer
+    //lorder -> Location to point/Id file
+    //porder -> Id to location file
+    unsigned npts_lorder, ndims_lorder, npts_porder, ndims_porder;
+    std::unique_ptr<char[]> lorder_data = std::make_unique<char[]>(npts_64);
+    std::unique_ptr<char[]> porder_data = std::make_unique<char[]>(npts_64);
+    if(lorder_file != std::string("") && porder_file != std::string("")){
+      reorder_data = true;
+      try {
+        diskann::load_bin<T>(lorder_file, lorder_data, npts_lorder, ndims_lorder);        
```

Comment:
T for this load_bin should be uint32_t as we need to load a binary file containing uint32_t data. same for next load_bin also